kivy/uix/actionbar.py

```diff
@@ -211,6 +211,17 @@ class ActionPrevious(BoxLayout, ActionItem):
        defaults to ''.
     '''
 
+    title_markup = BooleanProperty(False)
```

Comment:
Instead of `title_markup` use just `markup` like with the `Label` or `Button`. It'll be more obvious and user-friendly for devs to just try whether `markup` property "just works" when used somewhere, where obviously `Label` (or likes) is used. :)

Reply:
done